python27-app/module-main/taskqueue.py

```diff
@@ -613,6 +621,50 @@ def post(self):
         break
     self.response.set_status(200)
 
+
+class UpdateCallback(webapp2.RequestHandler):
+  @datastore.Transactional
+  def update_status(self, task_name):
+    key = datastore.Key.from_path('TaskName', task_name)
```

Comment:
Shouldn't `__task_name__` be used as a kind?
https://github.com/AppScale/appscale/pull/2777/files#diff-91eefc251646eea66197e5ffc86f3283R16

Reply:
The `__task_name__` entities are stored under the dashboard namespace and are an implementation detail for the push queue API. The `TaskName` entity here is just an entity used by the hawkeyepython27 app to keep track of when a side effect was made (indicating that the task was executed). I named it that way so that the entities would be associated with the test that checks task names.

Reply:
Thanks. I didn't realize that hawkeye has own `TaskName` model.